ui/server/src/main/scala/pl/touk/nussknacker/ui/api/ProcessesResources.scala

```diff
@@ -90,14 +92,12 @@ class ProcessesResources(val processRepository: FetchingProcessRepository,
             }
           }
         } ~ path("processesDetails") {
```

Comment:
Mayby better way is do GET /processes?names=[comma separeted names]? I think we should use only one endpoint with filter queries to fetching list of processes,

Reply:
/process endpoint serves different data than /processesDetails which returns more verbose representation of processes

Reply:
Okey! First: better name for this endpoint is /processes-datails/. Second: where are tests? :)